### DISCUSSION TITLE
How to use Microsoft Azure with Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by SJY051** [community/community#44643] [GitHub Education]
I tried to authenticate students on Microsoft Azure using Github Education, but it is not possible. How can I get Azure student benefits?

Basically, I've used most of the methods provided by Github and methods that can be searched for on Google. But as a result, I failed student authentication.

I tried to log in using my Github account, but it doesn't work with the following message.
![image](https://user-images.githubusercontent.com/68188784/213098476-a545ab7c-c3ca-4215-8346-cf38538c1c3e.png)

Also, when I googled, in the case of other people, there was a case where the Verification code was written in the part where the Azure benefits were listed on Github, but in my case it was not.

![image](https://user-images.githubusercontent.com/68188784/213099037-97048b85-25aa-439f-9752-d3acfc63ce61.png)

Therefore, it is of course impossible for me to log in using the verification code. 

How can I use Azure using my Github Student benefit?

(This article has been translated from Korean to English. I don't know English well, and I can't check the translation. Therefore, there may be a problem with the translation or some incorrect terms are used. Please forgive me.)

**Comment by K-54N7H05H** [comment]
I am also experiencing the same issue. Despite having an approved Github Student Developer Pack, it is unable to redeem Microsoft Azure for Students through Azure portal.

If anyone has found a workaround or way to redeem the benifit, it would be helpful to share

**Comment by Alhern** [comment]
Yep, still doesn't work.

**Comment by Aviratyash** [comment]
Anybody found solution for this please help me

**Comment by Andreprado20** [comment]
I'm also having the same trouble

**Comment by Rayen-trabelsi** [comment]
I have the same problem too

**Comment by ghost** [comment]
same problem.

**Comment by JoloanTsai** [comment]
same problem.

**Comment by LightYagami28** [comment]
In my case it worked perfectly I logged in with github

Edit:

I'm blocked because my academy it's not eligible from e-mail **damn**

**Reply by MarilynHb** [reply]
same here!!

**Comment by plambert684** [comment]
Bonjour

Je rencontre actuellement le même problème. J'ai ouvert un ticket Azure je proposerai une solution ici si mon problème se résous

**Comment by Goldsucher07** [comment]
sadly same here

**Comment by admdnlr** [comment]
same problem

**Comment by xylx521** [comment]
I am currently facing the same issue. Do you answer the question?

**Reply by JoloanTsai** [reply]
No , even after a year.

**Comment by Ayush-Kumar-web** [comment]
I am facing the same issue
I think it wants university email
unfortunately my university does not provide any edu mails to students

**Comment by y0geshx** [comment]
Same issues - 2024

I can't get verified by Microsoft 😢

**Comment by sneha20222026** [comment]
same issue getting @Kyanite0029

**Comment by jericho1050** [comment]
2024

same issue.

i sent. a ticket to microsoft azure support idk if their support can resolve this

**Comment by sneha20222026** [comment]
I know how to use it properly

**Reply by cristiandpto23** [reply]
hi. how?

**Comment by NourEddineRabouane** [comment]
![image](https://github.com/user-attachments/assets/616ef739-894e-460c-85eb-ced3510b3377)
-
-
Guys try to start from the green button in the picture , don't go for the sign in option . ( you will fill in some of your informations , and get the access )

**Reply by FizzWizZleDazzle** [reply]
I got approved for github education but my domain isn't registered with microsoft, and i don't see github as an option on there

**Reply by NourEddineRabouane** [reply]
i think you have to register on Microsoft using your university email, do not try to sign in from the GitHub website

**Comment by sneha20222026** [comment]
If you do not have university email, try requesting a code from Microsoft
for signing in to azure

**Comment by joseeverm** [comment]
same problem

**Comment by moneshgomo** [comment]
facing the same issue , i got the Github student developer pack but  Microsoft Azure is not allowing to claim

**Comment by nischhal-hub** [comment]
I linked my outlook account to the github account with student developer pack, then proceded to create account on azure with the same account via github, it worked.

**Reply by tuyuritio** [reply]
Same here!

**Reply by FizzWizZleDazzle** [reply]
Does your school use outlook, or that your personal outlook?

**Reply by nischhal-hub** [reply]
Used personal outlook account.

**Comment by VieCodess** [comment]
Anyone in 2026, I encountered same   problem but was able to fix it. I simply used a new personal account, signed in to microsoft and then i stated the sign up process for azure students which is an Academic Verification form where you fill your school and country. I added my schol email and they sent a veriification and Viola...I verified. Although they have some session issues so make sure you are in the location(country) where your school is.
Hope this helps
